src/services/swap_service.ts

```diff
@@ -10,6 +10,7 @@ import {
     SwapQuoter,
```

Comment:
@feuGeneA Don't we still need some sort of resolve on the issue of `_estimateGasOrThrowRevertErrorAsync` for when the taker address is the forwarder address?